pkg/model/config/config.go

```diff
@@ -147,34 +147,53 @@ func (c Config) resourceAPIVersionCompatible(verType, version string) bool {
 	return true
 }
 
-// GVK contains information about scaffolded resources
-type GVK struct {
+// Data contains information about scaffolded resources
+type Data struct {
 	Group   string `json:"group,omitempty"`
 	Version string `json:"version,omitempty"`
 	Kind    string `json:"kind,omitempty"`
 
-	// CRDVersion holds the CustomResourceDefinition API version used for the GVK.
+	// CRDVersion holds the CustomResourceDefinition API version used for the Data.
 	CRDVersion string `json:"crdVersion,omitempty"`
-	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the GVK.
+	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Data.
 	WebhookVersion string `json:"webhookVersion,omitempty"`
+
+	// Webhooks holds the Webhooks data
+	Webhooks Webhooks `json:"webhooks,omitempty"`
```

Comment:
I think that making this field a pointer and setting it to nil when all of them are false should not print the `webhooks: {}` that we were complaining about.
```suggestion
	Webhooks *Webhooks `json:"webhooks,omitempty"`
```

Reply:
Good excellent catcher and suggestion 👍